src/ServiceBus/ServiceBus.Autorest/custom/Set-AzServiceBusNamespaceV2.ps1

```diff
@@ -0,0 +1,253 @@
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Category('Body')]
+        [System.String]
+        ${Location},
+
+        [Parameter(HelpMessage = "Alternate name for namespace")]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Category('Body')]
+        [System.String]
+        ${AlternateName},
+        
+        [Parameter(HelpMessage = "This property disables SAS authentication for the Service Bus namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Category('Body')]
+        [System.Management.Automation.SwitchParameter]
+        ${DisableLocalAuth},
+		
+        [Parameter(HelpMessage = "Properties of KeyVault")]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Models.Api202201Preview.IKeyVaultProperties[]]
+        ${KeyVaultProperty},
+
+        [Parameter(HelpMessage = "Enable Infrastructure Encryption (Double Encryption)")]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Category('Body')]
+        [System.Management.Automation.SwitchParameter]
+        ${RequireInfrastructureEncryption},
+
+        [Parameter(HelpMessage = "Type of managed service identity.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Support.ManagedServiceIdentityType]
+        ${IdentityType},
+
+        [Parameter(HelpMessage = "Properties for User Assigned Identities")]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Category('Body')]
+        [System.Collections.Hashtable]
+        ${UserAssignedIdentity},
+
+        [Parameter(HelpMessage = "The minimum TLS version for the cluster to support, e.g. '1.2'")]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Support.TlsVersion]
+        ${MinimumTlsVersion},
+
+        [Parameter(HelpMessage = "This determines if traffic is allowed over public network. By default it is enabled.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Support.PublicNetworkAccess]
+        ${PublicNetworkAccess},
+
+        [Parameter(HelpMessage = "Name of this SKU.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Support.SkuName]
+        ${SkuName},
+
+        [Parameter(HelpMessage = "The Event Hubs throughput units for Basic or Standard tiers, where value should be 0 to 20 throughput units. The Event Hubs premium units for Premium tier, where value should be 0 to 10 premium units.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.Category('Body')]
+        [System.Int32]
+        ${SkuCapacity},
+
+        [Parameter(HelpMessage = "Value that indicates whether AutoInflate is enabled for ServiceBus namespace.")]
```

Comment:
@schaudhari6254888 , please remove MaximumThroughputUnits and EnableAutoInflate as ServiceBus does not support it.